src/commands/infra/create.ts

```diff
@@ -0,0 +1,137 @@
+import commander from "commander";
+import fs, { chmod } from "fs";
+import { logger } from "../../logger";
+import { exec } from "../../lib/shell";
+import path from "path";
+import { promisify } from "util";
+
+// Global vars that attain if spk infra init has been executed and the path to a bedrock template directory
+let initValidated: boolean = true; //Holder until spk infra init integration
```

Comment:
Should `const` instead of `let` when the variable is not going to be redefined; The TSLint extension for VSCode should highlight this